Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -349,6 +350,9 @@ public enum DefaultFunctionOpacityOptions {
 If verification fails, report a detailed counterexample for the first failing assertion (experimental).".TrimStart()) {
   };
 
+  public static readonly Option<string> BackendModuleName = new("--module-name",
```

Comment:
I'd prefer it if you override the `GetCommand` method in `GoBackend`, and add the option to the command created there, so it only shows up when using `translate go`.